web/src/components/Comment.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+
+const Comment = () => {
+  const [comments, setComments] = useState([]);
+  const [userId, setUserId] = useState(1); // Placeholder for user ID
+  const [content, setContent] = useState('');
+  const [message, setMessage] = useState('');
+
+  const backendUrl = "service/comments";
+
+  // Fetch comments when the component mounts
+  useEffect(() => {
+    fetchComments();
+  }, []);
+
+  const fetchComments = () => {
+    fetch(backendUrl)
+      .then(res => res.json())
+      .then(data => {
```

Comment:
You can avoid using a bunch of these ```.then``` statements by using async/await like in some of our other components, but I honestly don't care as long as it works LMAOOOO